cmd/cmd.go

```diff
@@ -90,7 +91,7 @@ func ExecInCoreSnap() {
 	// ensure we do not re-exec into an older version of snapd, look
 	// for info file and ignore version of core that do not yet have
 	// it
-	fullInfo := filepath.Join(corePath, "/usr/lib/snapd/info")
```

Comment:
Wait, this is not what I expected (the diff looked different before). So on Fedora the core path is `/var/lib/snapd/snap/core/current/` but then the `dirs.DistroLibExecDir` will contain `/usr/libexec/snapd`, I don't understand how this is supposed to work.